x/evm/keeper/evm.go

```diff
@@ -381,7 +381,7 @@ func (k *Keeper) ApplyMessageWithConfig(ctx cosmos.Context,
 
 	// if transaction is Aspect operational, short the circuit and skip the processes
 	if isAspectOpTx := asptypes.IsAspectContractAddr(msg.To); isAspectOpTx {
-		nativeContract := contract.NewAspectNativeContract(k.storeKey, k.getCtxByHeight, evm,
+		nativeContract := contract.NewAspectNativeContract(k.storeKey, evm,
 			ctx.BlockHeight, stateDB, k.logger)
```

Comment:
Why passing a function here, you can just pass the block height directly.

Reply:
1.   VerifyTx() requires this logic
2. The Context of the Aspect core layer cannot be resolved, and the Block Height cannot be obtained